packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/ModelFactoryProvider.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.Generator.CSharp.Providers
     internal class ModelFactoryProvider : TypeProvider
     {
         private const string ModelFactorySuffix = "ModelFactory";
-        private const string AdditionalRawDataParameterName = "serializedAdditionalRawData";
+        private const string AdditionalBinaryDataParameterName = "additionalBinaryDataProperties";
```

Comment:
Do we intend to rename the parameter in model factory methods? I think this is break from how it is named when generating using autorest.csharp, so just want to make sure this is intentional.

Reply:
ah that is a good point. Since all of the parameters in these factory methods are optional, this could be a breaking change. I suppose we should avoid that and I can make some changes to keep the parameter name the same for AP of binary data, unless we want to be consistent with the naming here.

Reply:
Resolving based on the previous comment.